.github/workflows/ci_cd.yml

```diff
@@ -486,8 +486,8 @@ jobs:
 
       - name: collectstatic
         run: |
-          just _all-up
           just api/collectstatic
+        # `api/collectstatic` uses `api/dj` which depends on `api/wait-up`
```

Comment:
This comment feels like a response to removing the `_all-up` line, I feel like you could just leave it out (and add a comment in the PR to explain the change) :sweat_smile:. The comment is confusing otherwise. Even with the context here I spent a few seconds trying to understand what the comment was documenting.